src/Integration/Binding/CFamilyProjectBinder.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
Missing copyright

Reply:
Every time! :(
Perhaps we change the quality profile and add the copyright rule?